src/pages/groupComparison/GroupComparisonMutationsTab.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import { action, computed, observable, makeObservable } from 'mobx';
+import autobind from 'autobind-decorator';
+import GroupComparisonStore from './GroupComparisonStore';
+import _ from 'lodash';
+import {
+    MUTATIONS_NOT_ENOUGH_GROUPS_MSG,
+    MUTATIONS_TOO_MANY_GROUPS_MSG,
+} from './GroupComparisonUtils';
+import { MakeMobxView } from 'shared/components/MobxView';
+import LoadingIndicator from 'shared/components/loadingIndicator/LoadingIndicator';
+import ErrorMessage from 'shared/components/ErrorMessage';
+import { LollipopGeneSelector } from './LollipopGeneSelector';
+import GroupComparisonMutationsTabPlot from './GroupComparisonMutationsTabPlot';
+
+interface IGroupComparisonMutationsTabProps {
+    store: GroupComparisonStore;
+}
+
+@observer
+export default class GroupComparisonMutationsTab extends React.Component<
+    IGroupComparisonMutationsTabProps,
+    {}
+> {
+    constructor(props: IGroupComparisonMutationsTabProps) {
+        super(props);
+    }
+
+    readonly selectorUI = MakeMobxView({
+        await: () => [this.props.store.availableGenes],
+        render: () => {
+            this.props.store.setDefaultGene();
+            return <LollipopGeneSelector store={this.props.store} />;
+        },
+    });
+
+    readonly tabUI = MakeMobxView({
+        await: () => [this.props.store.availableGenes],
+        render: () => {
+            // We only want 2 groups for our mirrored lollipop plot. Display message if not 2 groups
+            if (this.props.store.activeGroups.result!.length < 2) {
+                return <span>{MUTATIONS_NOT_ENOUGH_GROUPS_MSG}</span>;
+            } else if (this.props.store.activeGroups.result!.length > 2) {
+                return <span>{MUTATIONS_TOO_MANY_GROUPS_MSG}</span>;
+            }
+            return (
+                <>
+                    <div className="alert alert-info">
+                        Gene with highest frequency is displayed by default.
+                        Gene can be changed in the dropdown below.
+                    </div>
+                    {this.selectorUI.component}
+                    <GroupComparisonMutationsTabPlot
+                        store={this.props.store}
+                        mutations={_(this.props.store.mutationsByGroup.result!)
+                            .values()
+                            .flatten()
+                            .value()}
+                        filters={{
+                            groupFilters: _(
```

Comment:
_.(result).keys() and then you don't have to call .value() since you aren't actually chaining